### DISCUSSION TITLE
Copilot Nightly and CopilotLabs

### DISCUSSION THREAD

**Discussion by hindrik40** [community/community#48638] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi I have a payed license for CopilotNightly do I need a license for CopilotLabs also if a use them both?

**Comment by Akash1134** [comment]
Hi @hindrik40 , You will also need access to GitHub Copilot Labs. There is no waitlist but you will need to accept the [GitHub Next Pre-Release License Terms](https://github.com/githubnext/githubnext/blob/main/TERMS_AND_CONDITIONS.md).

If you're looking to use it with Copilot, install both the [GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and the [GitHub Copilot Labs extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-labs) for VS Code. GitHub Copilot Labs works only with VS Code right now.
